pallets/runtime/tests/src/confidential_asset_test.rs

```diff
@@ -1 +1,107 @@
+use super::{
+    storage::{register_keyring_account, TestStorage},
+    ExtBuilder,
+};
+use codec::Decode;
+use core::convert::TryFrom;
+use frame_support::assert_ok;
+use pallet_asset::{self as asset, AssetType, IdentifierType, SecurityToken};
+use pallet_confidential_asset as confidential_asset;
+use polymesh_primitives::Ticker;
+use test_client::AccountKeyring;
+type ConfidentialAsset = confidential_asset::Module<TestStorage>;
+type Origin = <TestStorage as frame_system::Trait>::Origin;
+use codec::Encode;
+use confidential_asset::EncryptedAssetIdWrapper;
+use cryptography::{
+    asset_proofs::{CommitmentWitness, ElgamalSecretKey},
+    mercat::{
+        account::{convert_asset_ids, AccountCreator},
+        AccountCreatorInitializer, EncryptedAmount, EncryptionKeys, SecAccount,
+    },
+    AssetId,
+};
+use curve25519_dalek::scalar::Scalar;
+use rand::{rngs::StdRng, SeedableRng};
 
+#[test]
+fn account_create_tx() {
+    ExtBuilder::default()
+        .build()
+        .execute_with(account_create_tx_valid_asset_id);
+}
+
+fn account_create_tx_valid_asset_id() {
+    // Simulating the case were issuers have registered some tickers and therefore the list of
+    // valid asset ids contains some values.
+    let issuer = AccountKeyring::Bob.public();
+
+    let valid_asset_ids: Vec<AssetId> = vec![1, 2, 3]
+        .iter()
+        .map(|id| AssetId::from(id.clone()))
```

Comment:
```suggestion
        .cloned()
        .map(AssetId::from)
```
Some eta-reduction is (almost) always nice. :)

Reply:
Neat!